src/components/data/lang/en.json

```diff
@@ -1,19 +1,19 @@
 {
     "key": "en",
-    "title": "Blubber",
-    "subtitle": "A docker-compose.yml builder for wikibase-docker",
+    "title": "wikibase-docker Config Wizard",
+    "subtitle": "A yaml builder for wikibase-docker",
```

Comment:
again, this should really be docker-compose specific.
yaml builder? what does that even mean? yaml + docker = nothing

Reply:
We could also have no subtitle, I guess.